README.md

```diff
@@ -69,7 +69,7 @@ This will download the repository to your current directory
 </ul>
 
     cd gonomics
-    go test ./…
+    go test …
```

Comment:
I just tried this on my installation and it gave me errors when I just did ... and not ./...